pages/common/pueue.md

```diff
@@ -0,0 +1,36 @@
+# pueue
+
+> Pueue is a command-line task management tool for sequential and parallel execution of long-running tasks.
+> More information: <https://github.com/Nukesor/pueue>.
+
+- Add a command to the task list:
+
+`pueue add -- {{tr -u a-z A-Z}}`
```

Comment:
Why `{{tr -u a-z A-Z}}` and not `{{command}}`? I think the latter would be more clear in this instance.

Reply:
I picked that because it's easy to demonstrate the functionality of `pueue send` and `pueue logs`, as one operates on stdin and the other on stdout. `-u` ensures the output is always unbuffered, so the stdin would immediately be processed and sent to stdout.  Would you still like it changed?

Reply:
It also demostrates command flags and multiple arguments.

Reply:
I kinda agree with @sbrl here. I feel like it doesn't get the the point of why this tool exists, which I would say is running commands that take a long time in the background. Maybe a better example is `tar` if we don't want ``{{command}}``? I feel like `pueue send`'s description is clear enough that we don't need the command here be based around stdin.

Reply:
Hmm...I forgot to mention, another reason I picked `tr` is because it runs forever, until you stop it.  If you still think so, I'll change it.  Please tell me.

Reply:
It's just my two cents, @Nukesor probably should have the final say as the creator of `pueue`

Reply:
Just adding this, under the [A few possible applications](https://github.com/Nukesor/pueue#why-should-i-use-it) they are all commands that take a while. I feel like `tldr` should have examples that show off what the command is useful for but again, @Nukesor probably knows best.

Reply:
I think `{{command}}` would be the best solution here, as it's the most neutral. But I do appreciate the point about using a long-running command.

Reply:
Thinking about it, `{{command}}` is probably better, since it indicates that any command can be added.

I didn't think that it's this hard to get as much information as possible into two lines of documentation, without confusing the user :D.

Especially, since adding commands in Pueue can actually be quite tricky/confusing, as soon if any characters need escaping.
Fittingly, there was an [issue](https://github.com/Nukesor/pueue/issues/179) just a few days ago, regarding escaping on `pueue add`.

I thought about adding quotation marks around the command `pueue add "{{command}}"`, but this might lead to more confusion than it might clarify things. What do you think about this?

Reply:
:+1: for `pueue add "{{command}}"`.  @sbrl?

Reply:
How about creating separate pages for subcommand (like with git: `git-add.md`, `git-clone.md`). some subcommands deserve their own pages (e.g. `pueue add`) while others don't (e.g. `pueue clean`). 

> Thinking about it, {{command}} is probably better, since it indicates that any command can be added.

If we have `pueue-add.md`. the first example could show the generic `pueue add -- {{command}}`...

> Just adding this, under the A few possible applications they are all commands that take a while. I feel like tldr should have examples that show off what the command is useful for 

Next examples would show something like:

```
# pueue add

> Enqueue a task for execution

- add {{command}} to the default queue.
 
`pueue add {{command}}`

- pass flags and arguments {{command}} while adding.
 
`pueue add -- {{command --Arg1 -f}}`

- add command to a [g]roup, see "tldr pueue-group" to operate groups.

`pueue add -g "CPU_intensive" -- ffmpeg -i input.mp4 frame_%d.png`

- add command with a [l]abel after a [d]elay has passed, see "tldr pueue-enqueue" for accepted formats.

`pueue add -l "compressing large file" -d "wednesday 10:30pm" "7z a compressed.7z large_file.xml"`
```
`pueue.md` would look like:

```
  Check pueue version:

      pueue --version

  Show general help and available subcommands:

      pueue --help

  Show more example on a pueue subcommand (like `add`, `parallel`, etc.):

      tldr pueue-subcommand
```
If you approve this I can add commits on top of `bl-ue/add-pueue` branch.

Reply:
@she3o that sounds great!  I'd love for you to finish this! However, I think it would be better for me to finish this PR (I'll go with `"{{command}}"` since that's the consensus) and you to open separate PR(s) for GitHub UI clarity and also so you get your own contributions ;)

Reply:
I'll go with "{{command}}" since that's the consensus
```suggestion
`pueue add -- "{{command}}"`
```